test/groupby_operations.jl

```diff
@@ -0,0 +1,2623 @@
+@testset "groupby and combine(::Function, ::GroupedDataFrame)" begin
```

Comment:
Maybe call the file "grouped_operations.jl", as "groupby" is the name of a particular function?

Reply:
OK - changed.